### ISSUE TITLE
carry timestamp in SMS message

### ISSUE THREAD

**Issue by olafhering** [Seva-coder/Finder#8]
Recently I had the case where the "location request" SMS was delivered hours later to the other device.

It would be nice if the SMS messages  contain a timestamp from the device that sends the request, as well as a timestamp from the device that responds to a request. That way the location info would be more reliable.

If such timestamp should be UTC or localtime has to be decided. No idea what APIs are offered by Android.

**Comment by n005** [comment]
Yes, good idea, for the "track", it should avoid roll-back messages.

**Comment by Seva-coder** [comment]
we can use unix time for this purpose (sms have enough place for it). Yes this information in response will be useful.
Time in "tracking" now is local, so it should be changed too
